mflowgen/utils/__init__.py

```diff
@@ -1,4 +1,7 @@
+# Note '__init__.py' not required for python 3.3 and above, see
+# https://docs.python.org/3/reference/import.html#regular-packages
+
 from mflowgen.utils.helpers import get_top_dir, get_files_in_dir
 from mflowgen.utils.helpers import bold, yellow, red, green
 from mflowgen.utils.helpers import read_yaml, write_yaml
-
+from mflowgen.utils.helpers import stamp
```

Comment:
This of course has nothing to do with the sorted lists...I just noticed that one of the methods had been left out of utils `__init__.py` and decided to make things consistent...

Reply:
Fair enough.